hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpCommandProcessor.java

```diff
@@ -213,12 +214,22 @@ private boolean authenticate(@Nonnull HttpPostCommand command,
         }
         String decodedPass = pass != null ? URLDecoder.decode(pass, "UTF-8") : null;
         UsernamePasswordCredentials credentials = new UsernamePasswordCredentials(decodedName, decodedPass);
+        Boolean passed = Boolean.FALSE;
```

Comment:
Why a Boolean?

Reply:
It avoids boxing in the `addParameter` call.

Reply:
Afaik no new object is made if you have a boolean that is being boxed. So I don't think there is any difference from a performance perspective. Anyhow. It isn't very important.